### ISSUE TITLE
Accessibility Audit for Jupyter Notebook

### ISSUE THREAD

**Issue by eileencho** [jupyter/accessibility#7]
Hello, I’m a student at NYU's Center for Data Science, working with @VickySteeves this summer on an accessibility audit for Jupyter Notebooks. I was trained by NYU IT's Accessibility Office to be able to do a full audit. I've gone through Jupyter Notebook version 6.0.0, using notebooks from the [Data 8](https://github.com/data-8/materials-sp19) course materials as examples, and VoiceOver on Mac for screen reader testing. Below I’ve included some critical accessibility issues as starting points, along with a [Google Doc for more details](https://docs.google.com/document/d/1q_7LiqLX2WGJZ0UHNWDFafw75McZgLp9v4Ifdzn2FYE/edit?usp=sharing). I also have a spreadsheet of the [complete audit](https://docs.google.com/spreadsheets/d/1iCiEExo4VDMOwAYtBTCdh2chyx394-1r6YwzmRILF9E/edit?usp=sharing) I performed, showing the pages I tested, which areas did not pass WCAG 2.0. Each bolded point under the “Test Procedure” column links to the corresponding section on [WCAG 2.0](https://www.w3.org/TR/WCAG20/) for reference. I also referenced [WAI-ARIA Authoring Practices](https://www.w3.org/TR/wai-aria-practices/) for examples on design patterns.

- Accessible name missing for folder/file icons
- Status changes are not announced for assistive technologies
- Graphs generated by code need alternative way to convey information
- Menu items are difficult to access with keyboard
- Keyboard trap
- Code editor mode or Command mode information not conveyed
